src/components/SelectionList/BaseListItem.tsx

```diff
@@ -100,20 +99,22 @@ function BaseListItem({
                             </View>
                         </View>
                     )}
+
                     <ListItem
                         item={item}
                         textStyles={[
                             styles.optionDisplayName,
                             isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText,
-                            styles.sidebarLinkTextBold,
+                            isUserItem ? styles.sidebarLinkTextBold : null,
```

Comment:
This is causing the bug I mentioned in the PR comment.

Reply:
@allroundexperts I am not sure if text in all selectors should be bolded tbh (I think it was not bolded before)

Reply:
Hm... I am comparing it with staging. Can you please get a confirmation about this in the open source channel?

Reply:
It looks strange imo 😄

Reply:
True that. Looks so to me as well. But we should not do it ourselves just because it looks strange. Let's get it confirmed!

Reply:
Probably should be bolded according to [comment on open source channel](https://expensify.slack.com/archives/C01GTK53T8Q/p1706041138735109?thread_ts=1706040628.308479&cid=C01GTK53T8Q)👀

Reply:
@allroundexperts fixed and main merged!

Reply:
Fixing typescript error